organization/privacy.md

```diff
@@ -8,11 +8,12 @@ redirect_from:
 
 # Privacy
 
-The Foundation strongly supports individuals' right to privacy. For this reason we don't track individual users on any of our websites but count traffic for analytics.
+The Foundation strongly supports individuals' right to privacy. 
```

Comment:
I think this needs to be further explained. If we really "strongly support individuals' right to privacy", we shouldn't track individual users. Removing this sentence without changing the first sentence seems contradictory, unless we have an explanation somewhere to why we feel tracking individuals is supporting that right.

Reply:
@Ainali this should be resolved now.